### PR TITLE
INFRA-2133: Don't send metrics when values are empty

### PR DESCRIPTION
**Pull request by Backscratcher**
I think this packages needs more changes to humanize it a little bit more. Tests are hard to write and code is not compatible with python3.10. I will create a task to fix it and then change stuff here or completely abandon this package.

### CODE REVIEW

File: django_celery_monitor/cloudwatch_camera.py [hunk 1]
```diff
@@ -77,17 +77,19 @@ def prepare_metrics(self):
...
-                self.add(
-                    Metric(
-                        name="QueueWaitingTasks",
-                        dimensions={
-                            "QueueName": queue.name,
-                            "Environment": environment,
-                        },
-                        unit="Count",
-                        value=self._check_queue(connection, queue.name)
+                value = self._check_queue(connection, queue.name)
```
Comment (4fuss) [inline django_celery_monitor/cloudwatch_camera.py:80]: Where this metric is coming from? Is it read from broker (redis)?

**Review by 4fuss — COMMENTED** [review]
Looks good, question just for my comprehension. Where `QueueWaitingTask` value is get from?

File: django_celery_monitor/cloudwatch_camera.py (hunk 1 above)
Comment (Backscratcher) [reply django_celery_monitor/cloudwatch_camera.py:80]: yes :)

**Comment by Backscratcher** [comment]
> Looks good, question just for my comprehension. Where `QueueWaitingTask` value is get from?

Like I confirmed already, this is taken from broker - redis.

**Review by 4fuss — APPROVED** [review]

**Review by Kaniabi — APPROVED** [review]

**Pull request merged**